outputs/hamronized/abricate.argannot.tsv

```diff
@@ -801,7 +801,7 @@ GMGC10.002_739_964.BLA	GMGC10.95nr_block_0020	(Bla)blaVHH-1	(Bla)blaVHH-1	argann
 GMGC10.002_743_408.YBXI	GMGC10.95nr_block_0020	(Bla)blaOXA-228	(Bla)blaOXA-228	argannot	2021-Mar-27	JQ422053:100-930	abricate	abricate 1.0.1	80.38			47	846					+		96.27									ARO:3001691	ARO:0000020,ARO:3000008	ARO:3000007,ARO:3000007
 GMGC10.002_823_424.AADK	GMGC10.95nr_block_0020	(AGly)aadK-Pm	(AGly)aadK-Pm	argannot	2021-Mar-27	CP002869:c4442322-4441435	abricate	abricate 1.0.1	82.9			2	849					+		95.5									ARO:3002627	ARO:0000040	ARO:0000016
 GMGC10.003_659_233.BLA	GMGC10.95nr_block_0020	(Bla)blaVHH-1	(Bla)blaVHH-1	argannot	2021-Mar-27	AF217649:134-985	abricate	abricate 1.0.1	82.04			1	851					+		99.77									ARO:3007003	ARO:3000637	ARO:3000007
-GMGC10.005_099_684.ERMA	GMGC10.95nr_block_0020	(MLS)erm(K)	(MLS)erm(K)	argannot	2021-Mar-27	NG_047831:101-955	abricate	abricate 1.0.1	100.0			1	855					+		100.0									ARO:3004364	ARO:3000053	ARO:3000053
+GMGC10.005_099_684.ERMA	GMGC10.95nr_block_0020	(MLS)erm(K)	(MLS)erm(K)	argannot	2021-Mar-27	NG_047831:101-955	abricate	abricate 1.0.1	100.0			1	855					+		100.0									ARO:3004643	ARO:0000006	ARO:0000000
```

Comment:
Can you comment on this on the commit as well?

How many argannot genes changed ARO annotation?